experimental/devices/epd/epd.go

```diff
@@ -0,0 +1,463 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package epd
+
+import (
+	"errors"
+	"fmt"
+	"image"
+	"image/color"
+	"image/draw"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/display"
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/gpio/gpioreg"
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/spi"
+	"periph.io/x/periph/devices/ssd1306/image1bit"
+)
+
+// Raspberry pi hat pins
+const (
+	ResetPin       = "17"
+	DataCommandPin = "25"
+	ChipSelectPin  = "8"
+	BusyStatePin   = "24"
+)
+
+// EPD commands
+const (
```

Comment:
Please do not export them, the user is not expected to send raw commands to the device.